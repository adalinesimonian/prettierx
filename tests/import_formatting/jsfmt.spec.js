run_spec(__dirname, ["babel", "babel-flow", "babel-ts"], {
  arrowParens: "avoid", // added to preserve snapshot order
  importFormatting: "oneline",
});
run_spec(__dirname, ["babel", "babel-flow", "babel-ts"], {
  importFormatting: "auto",
  trailingComma: "none",
});
