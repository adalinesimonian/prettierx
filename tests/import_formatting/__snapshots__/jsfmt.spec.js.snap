// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`long_import.js - {"arrowParens":"avoid","importFormatting":"oneline"} format 1`] = `
====================================options=====================================
arrowParens: "avoid"
importFormatting: "oneline"
parsers: ["babel", "babel-flow", "babel-ts"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
import abc, { Lorem, ipsum, dolor, sit, amet, consectetur, adipiscing, elit, sed, eiusmod, tempor, incididunt, ut, labore, et, magna } from "some-module";
import { Lorem, ipsum, dolor, sit, amet, consectetur, adipiscing, elit, sed, eiusmod, tempor, incididunt, ut, labore, et, magna } from "some-module";

=====================================output=====================================
import abc, { Lorem, ipsum, dolor, sit, amet, consectetur, adipiscing, elit, sed, eiusmod, tempor, incididunt, ut, labore, et, magna } from "some-module";
import { Lorem, ipsum, dolor, sit, amet, consectetur, adipiscing, elit, sed, eiusmod, tempor, incididunt, ut, labore, et, magna } from "some-module";

================================================================================
`;

exports[`long_import.js - {"importFormatting":"auto","trailingComma":"none"} format 1`] = `
====================================options=====================================
importFormatting: "auto"
parsers: ["babel", "babel-flow", "babel-ts"]
printWidth: 80
trailingComma: "none"
                                                                                | printWidth
=====================================input======================================
import abc, { Lorem, ipsum, dolor, sit, amet, consectetur, adipiscing, elit, sed, eiusmod, tempor, incididunt, ut, labore, et, magna } from "some-module";
import { Lorem, ipsum, dolor, sit, amet, consectetur, adipiscing, elit, sed, eiusmod, tempor, incididunt, ut, labore, et, magna } from "some-module";

=====================================output=====================================
import abc, {
  Lorem,
  ipsum,
  dolor,
  sit,
  amet,
  consectetur,
  adipiscing,
  elit,
  sed,
  eiusmod,
  tempor,
  incididunt,
  ut,
  labore,
  et,
  magna
} from "some-module";
import {
  Lorem,
  ipsum,
  dolor,
  sit,
  amet,
  consectetur,
  adipiscing,
  elit,
  sed,
  eiusmod,
  tempor,
  incididunt,
  ut,
  labore,
  et,
  magna
} from "some-module";

================================================================================
`;
